src/plugins/console/server/lib/spec_definitions/js/aggregations.ts

```diff
@@ -320,13 +320,40 @@ const rules = {
     geohash_grid: {
       __template: {
         field: '',
-        precision: 3,
+        precision: 5,
```

Comment:
I also changed this to the [default setting](https://www.elastic.co/guide/en/elasticsearch/reference/master/search-aggregations-bucket-geohashgrid-aggregation.html#_options_3).